src/app/App.tsx

```diff
@@ -98,11 +99,19 @@ export class App extends React.Component<Props> {
 
     Linking.addEventListener('url', this.handleOpenURL)
 
+    dynamicLinks().onLink(({ url }) => this.handleOpenURL(url))
+
     const url = await Linking.getInitialURL()
     if (url) {
       await this.handleOpenURL({ url })
     }
 
+    const firebaseUrl = await dynamicLinks().getInitialLink()
+
+    if (firebaseUrl) {
+      await this.handleOpenURL({ url: firebaseUrl.url })
+    }
```

Comment:
We should wrap these under the `FIREBASE_ENABLED` flag.

What happens if firebase hasn't been initialized? There may be a race condition if the saga doing the firebase init hasn't finished by the time this is called.

Reply:
Ah yes - I was going to ask about that. I'll add the flag and test to see.

Reply:
@jeanregisser let me know if you're satisfied with this solution.